src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -54,43 +58,76 @@
 # % options: N-S,E-W
 # % answer: N-S
 # %end
+
 # %option G_OPT_R_OUTPUT
 # % key: rmedian
 # % description: Raster map of median lines
 # % required: no
 # % guisection: Optional
 # %end
+
 # %option G_OPT_V_OUTPUT
 # % key: vmedian
 # % description: Vector map of median lines
 # % required: no
 # % guisection: Optional
 # %end
+
+# %option G_OPT_V_OUTPUT
+# % key: vcategories
+# % description: Vector map of areas
+# % required: no
+# % guisection: Optional
+# %end
+
 # %option G_OPT_V_OUTPUT
 # % key: transects
 # % description: Vector map of complete transects
 # % required: no
 # % guisection: Optional
 # %end
+
 # %option G_OPT_V_OUTPUT
 # % key: itransects
 # % description: Vector map of inner transects
 # % required: no
 # % guisection: Optional
 # %end
+
 # %option G_OPT_F_OUTPUT
 # % key: csvfilename
 # % description: CSV output file
 # % required: no
 # % guisection: Optional
 # %end
 
+# %option
+# % key: metric
+# % type: string
+# % description: Determines how transect spacing is measured
+# % multiple: no
+# % options: straight, along
+# % descriptions: straight;Straight distance between transect points;along;Distance along the line (see v.transect for details)
+# % answer: straight
+# %end
+
+# %option
+# % key: transect_perpendicular
+# % type: string
+# % description: Determines which line is the transect perpendicular to
+# % multiple: no
+# % options: trend, line
+# % descriptions: trend;Perpendicular to the line connecting transect points;line;Perpendicular to the particular segment of the original line  (see v.transect for details)
+# % answer: trend
```

Comment:
Does the last parser section need to finish with an `%end` too?

Reply:
Yes, thanks for catching that, will correct.